src/ocm/components/clusterConfiguration/staticIp/components/YamlView/yamlViewValidationSchema.tsx

```diff
@@ -57,7 +57,8 @@ const macInterfaceMapValidationSchema = Yup.array<MacInterfaceMap>().of(
       .concat(getUniqueValidationSchema(getAllMacAddresses)),
     logicalNicName: Yup.string()
       .required(requiredMsg)
-      .concat(getUniqueValidationSchema(getInterfaceNamesInCurrentHost)),
+      .concat(getUniqueValidationSchema(getInterfaceNamesInCurrentHost))
+      .max(15, 'Interface name must be 15 characters at most.'),
```

Comment:
Can an interface name contain any of these characters: `!@#$%^&*()` ????

Reply:
@osherdp any chance we can use your help with regards to the validations that need to be performed on the interface name?

Reply:
Seems like it can really be anything: https://unix.stackexchange.com/questions/451368/allowed-chars-in-linux-network-interface-names
I mean, there are conventions and also auto-generated names by linux that are the more likely to get, but we might have different cases in general